core/src/main/java/nl/nn/adapterframework/jdbc/dbms/OracleToH2Translator.java

```diff
@@ -34,6 +34,15 @@
  * (<code>url=jdbc:h2:~/test;MODE=Oracle</code>). Missing compatibility is
  * covered in this class.
  * </p>
+ * <p>
+ * Note 1: Date functions in H2 use the value of property 'user.language'. So be
+ * sure it has the correct value
```

Comment:
Wat is dan die correct value? Waar moet die mee matchen?

Reply:
ik kan de foutsituatie niet meer herhalen. Opmerking uit de javadoc verwijderd